source/materials/OpticalMaterialProperties.cc

```diff
@@ -1666,33 +1656,21 @@ namespace opticalprops {
   {
       G4MaterialPropertiesTable* mpt = new G4MaterialPropertiesTable();
 
-      // REFLECTIVITY
-      std::vector<G4double> ENERGIES = {
-              optPhotMinE_, 7.29 * eV,  optPhotMaxE_
-      };
-
-      // We assume a default flat reflectivity of the stainless steel of 20%
-      // Measurements may be required to update these values to something 
-      // more realistic. 
-      std::vector<G4double> REFLECTIVITY = { 0.20,0.20,0.20};
-      // std::vector<G4double> REFLECTIVITY = { 0.00,0.00,0.00};
-
-      // REFLEXION BEHAVIOR
-      std::vector<G4double> ENERGIES_2    = {optPhotMinE_, optPhotMaxE_};
-      // Specular reflection about the normal to a microfacet.
-      // Such a vector is chosen according to a gaussian distribution with
-      // sigma = SigmaAlhpa (in rad) and centered in the average normal.
-      std::vector<G4double> specularlobe  = {0., 0.};
-      // specular reflection about the average normal
-      std::vector<G4double> specularspike = {0., 0.};
-      // 180 degrees reflection.
-      std::vector<G4double> backscatter   = {0., 0.};
-      // 1 - the sum of these three last parameters is the percentage of Lambertian reflection
-
-      mpt->AddProperty("SPECULARLOBECONSTANT", ENERGIES_2, specularlobe);
-      mpt->AddProperty("SPECULARSPIKECONSTANT",ENERGIES_2, specularspike);
-      mpt->AddProperty("BACKSCATTERCONSTANT",  ENERGIES_2, backscatter);
-      mpt->AddProperty("REFLECTIVITY", ENERGIES, REFLECTIVITY);
+      // Reflectivity
+      std::vector<G4double> refl_energies = {
+              optPhotMinE_, hc_ / (500. * nm), hc_ / (350. * nm),
+                            hc_ / (300. * nm), hc_ / (170. * nm),  optPhotMaxE_ };
+
+      // We assume a reflectivity of the stainless steel of 20% at VUV
```

Comment:
Regarding the reflectivities of steel and copper, so far we have assumed zero all over the place. I don't have a feeling of how much this modification would change the total collected light, but maybe we could make a small production of Kr to compare and be aware of the implications.
I would like to hear @jmalbos (and maybe @jmunozv) opinions here.

Reply:
It could also be a different PR from this one, at least regarding copper's reflectivity, which is not directly related to the grids. Also, you've added a reflective surface to the copper of the EP, but why not to the TP copper plate, also?

Reply:
Yes I can move the reflectivities of copper to a different one if that is preferred. As for why only the EP, the TP should be fully covered with the DICE boards whereas there is some exposed copper surface on the EP.